test/e2e/service.go

```diff
@@ -1151,6 +1171,31 @@ func testReachable(ip string, port int) bool {
 	return testReachableInTime(ip, port, podStartTimeout)
 }
 
+func testNetcatReachable(ip string, port int) {
+	con, err := net.Dial("udp", ip+":"+string(port))
+	if err != nil {
+		Failf("Failed to connect to: %s:%d (%s)", ip, port, err.Error())
+	}
+
+	_, err = con.Write([]byte("\n"))
+	if err != nil {
+		Failf("Failed to send newline: %s", err.Error())
+	}
+
+	var buf []byte = make([]byte, len("SUCCESS")+1)
```

Comment:
I have been running it in production to LB two separate UDP SSL VPNs with no issues.  To be fair though, this is just about setting up the LB to point the traffic correctly. Once that is done, there isn't a difference as far as capability from the normal k8s LB.  Any dealings with performance characteristics (beyond setup and teardown) is more a factor of GCE than k8s